src/plugins/rest_plugin/src/rest_plugin.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import pydantic
+import requests
+from dstack._internal.core.errors import ServerError
+from dstack._internal.core.models.fleets import FleetSpec
+from dstack._internal.core.models.gateways import GatewaySpec
+from dstack._internal.core.models.volumes import VolumeSpec
+from dstack.plugins import ApplyPolicy, Plugin, RunSpec, get_plugin_logger
+from dstack.plugins._models import ApplySpec
+
+logger = get_plugin_logger(__name__)
+
+PLUGIN_SERVICE_URI_ENV_VAR_NAME = "DSTACK_PLUGIN_SERVICE_URI"
```

Comment:
If the goal of this plugin is to allow enabling http-based plugins that don't require installing python packages, then there should be a way to configure multiple endpoints using this plugin, which is not possible with one env variable. We could probably support this once we allow specifying plugin options via yaml, and users will be able to specify rest_plugin multiple times (also need to support multiple registrations of the same plugin):

```
plugins:
  - rest_plugin:
      uri: http://localhost:8004
  - rest_plugin:
      uri: http://localhost:8005
```

This is just a notice for future development, not necessary applicable to the first plugin version.